ui/src/views/infra/changeStoragePoolScope.vue

```diff
@@ -0,0 +1,226 @@
+// Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Should the file be named `ChangeStoragePoolScope.vue` (uppercase `C`) for consistency?

Reply:
Renamed the file